abi/sector.go

```diff
@@ -97,6 +97,12 @@ const (
 	RegisteredPoStProof_StackedDrgWindow64GiBV1   = RegisteredPoStProof(9)
 )
 
+type RegisteredAggregationProof int64
+
+const (
+	RegisteredAggregationProof_SnarkPackV1 = RegisteredAggregationProof(0)
```

Comment:
I drafted this type here: https://github.com/filecoin-project/rust-filecoin-proofs-api/blob/feat-aggregation/src/registry.rs#L27

I'll rename mine for consistency though.